gravitee-apim-portal-webui/src/app/components/gv-documentation/gv-documentation.component.ts

```diff
@@ -189,7 +189,9 @@ export class GvDocumentationComponent implements OnInit, AfterViewInit {
 
   selectPage(pageId: string) {
     const pageToDisplay = this._pages.find(page => page.id === pageId);
-    GvDocumentationComponent.reset(this.treeMenu?.nativeElement);
+    setTimeout(() => {
```

Comment:
I always try to avoid these hacks. This hack modifies the execution thread of the JS task to place it at the end of the fill. In some cases, this solves problems, but it can also create new ones.

but since we've got lit + angular + ..., and the portal is no longer a project for the future. That doesn't seem to me to be a problem, as long as it fixes the issue.

Reply:
I found this hack in a few places in the portal components, the problem here is that treeMenu was undefined on the reset, so left sidebar was not displayed properly